src/test/java/uk/gov/hmcts/reform/ccd/document/am/feign/CaseDocumentClientTest.java

```diff
@@ -115,7 +124,66 @@ void shouldSuccessfullyUploadDocuments() throws JsonProcessingException {
         stubForUpload(request, mockResponse);
 
         UploadResponse uploadResponse = caseDocumentClient.uploadDocuments(
-            AUTHORISATION_VALUE, SERVICE_AUTHORISATION_VALUE, CASE_TYPE_ID, JURISDICTION, List.of(multipartFile));
+            AUTHORISATION_VALUE,
+            SERVICE_AUTHORISATION_VALUE,
+            CASE_TYPE_ID,
+            JURISDICTION, List.of(multipartFile));
+
+        List<Document> documents = uploadResponse.getDocuments();
+
+        assertThat(documents)
+            .hasSize(1)
+            .first()
+            .satisfies(document -> {
+                        assertThat(document.classification).isEqualTo(Classification.PUBLIC);
+                        assertThat(document.size).isEqualTo(1000);
+                        assertThat(document.mimeType).isEqualTo(MIME_TYPE);
+                        assertThat(document.originalDocumentName).isEqualTo(ORIGINAL_DOCUMENT_NAME);
+                        assertThat(document.hashToken).isEqualTo(HASH_TOKEN);
+                        assertThat(document.links.binary.href).isEqualTo(BINARY_VALUE);
+                        assertThat(document.links.self.href).isEqualTo(SELF_VALUE);
+                        assertThat(document.ttl).isEqualTo(ttl);
+                    }
+            );
+    }
+
+    @Test
+    void shouldSuccessfullyUploadDocumentsUsingMultipartFileImplementation() throws IOException {
```

Comment:
Some duplication, could have private method

```
private void assertDocumentUpload(MultipartFile multipartFile) throws JsonProcessingException {
        DocumentUploadRequest request = new DocumentUploadRequest(Classification.RESTRICTED.name(),
                CASE_TYPE_ID, JURISDICTION, List.of(multipartFile));

        Date ttl = new Date();

        Document.Links links = getLinks();

        Document mockDocument = Document.builder()
                .classification(PUBLIC)
                .hashToken(HASH_TOKEN)
                .mimeType(MIME_TYPE)
                .size(1000)
                .originalDocumentName(ORIGINAL_DOCUMENT_NAME)
                .ttl(ttl)
                .links(links)
                .build();

        UploadResponse mockResponse = new UploadResponse(List.of(mockDocument));

        stubForUpload(request, mockResponse);

        UploadResponse uploadResponse = caseDocumentClient.uploadDocuments(
                AUTHORISATION_VALUE,
                SERVICE_AUTHORISATION_VALUE,
                CASE_TYPE_ID,
                JURISDICTION, List.of(multipartFile));

        List<Document> documents = uploadResponse.getDocuments();

        assertThat(documents)
                .hasSize(1)
                .first()
                .satisfies(document -> {
                            assertThat(document.classification).isEqualTo(Classification.PUBLIC);
                            assertThat(document.size).isEqualTo(1000);
                            assertThat(document.mimeType).isEqualTo(MIME_TYPE);
                            assertThat(document.originalDocumentName).isEqualTo(ORIGINAL_DOCUMENT_NAME);
                            assertThat(document.hashToken).isEqualTo(HASH_TOKEN);
                            assertThat(document.links.binary.href).isEqualTo(BINARY_VALUE);
                            assertThat(document.links.self.href).isEqualTo(SELF_VALUE);
                            assertThat(document.ttl).isEqualTo(ttl);
                        }
                );
    }
    
```
   which would mean test methods would then look like this
   
 ```
    @Test
    void shouldSuccessfullyUploadDocumentsUsingMockMultiPartFile() throws IOException {

        MultipartFile multipartFile = new MockMultipartFile("file1",
                "test.png",
                "application/octet-stream",
                "someBytes".getBytes());

        assertDocumentUpload(multipartFile);
    }

    @Test
    void shouldSuccessfullyUploadDocumentsUsingMultipartFileImplementation() throws IOException {
        String url = "src/test/resources/validation.txt";
        File file = new File(url);
        FileItemFactory factory = new DiskFileItemFactory(16, file.getParentFile());
        FileItem item = factory.createItem(file.getName(),"text/plain",true, file.getName());

        Path path = Paths.get(url);
        Files.copy(path, item.getOutputStream());
        MultipartFile multipartFile = new CommonsMultipartFile(item);

        assertDocumentUpload(multipartFile);
    }

Reply:
fixed in commit [4f54246](https://github.com/hmcts/ccd-case-document-am-client/pull/19/commits/4f5424666df4393602af6fed2d85cfa235659cde)